src/prefect/deployments/schedules.py

```diff
@@ -47,7 +45,7 @@ def normalize_to_minimal_deployment_schedules(
             else:
                 raise ValueError(
                     "Invalid schedule provided. Must be a schedule object, a dict,"
-                    " or a MinimalDeploymentSchedule."
+                    "or a DeploymentScheduleCreate"
```

Comment:
```suggestion
                    "or a `DeploymentScheduleCreate` object"
```
nit, maybe, for clarify